src/test/java/teammates/test/pageobjects/InstructorStudentListPage.java

```diff
@@ -29,6 +38,31 @@ public InstructorStudentListPage(Browser browser) {
     protected boolean containsExpectedPageContents() {
         return getPageSource().contains("<h1>Student List</h1>");
     }
+    
+    public void toggleShowEmailCheckbox() {
+        click(showEmailCheckbox);
+    }
+    
+    public void clickCopyEmailButton() {
+        click(copyEmailButton);
+    }
+    
+    public boolean isCopyEmailButtonVisible() {
+        return copyEmailButton.isDisplayed();
+    }
+    
+    public boolean isCopyEmailPopoverVisible() {
+        String xpath = "//button[@id='copy_email_button']/following-sibling::div[contains(@class, 'popover')]";
```

Comment:
I want to be more specific about the popover related to the 'copy_email_button' (using `following-sibling`). If I use the css class selector like `class="popover"`, there might be other places in the page using `popover` and we may not get the correct one.